lib/shared/config/customize.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+var config = require('./');
+var merge = require('lodash.merge');
+var isString = require('lodash.isstring');
+
+// If project dir and cwd are same, the gulprc file in the directory is loaded
+// twice continuously. This function is to avoid it.
+var lastLoadedFile = null;
+function isNotLastLoaded(file) {
+  if (lastLoadedFile === file) {
+    return false;
+  }
+  lastLoadedFile = file;
+  return true;
+}
+
+module.exports = function(configFiles) {
+  configFiles.filter(isString).filter(isNotLastLoaded).forEach(function(file) {
+    merge(config, require(file));
```

Comment:
Yeah, it should be passed but I'm not sure if it should be another argument or merged with `opts` or `env` that are already passed.  Eventually we will merge the flags with `opts`.  What do you think?